packages/web/src/components/search/DataSearch.js

```diff
@@ -1065,8 +1069,27 @@ class DataSearch extends Component {
 															highlightedIndex,
 															index,
 														),
+														justifyContent: 'flex-start',
 													}}
 												>
+													<div style={{ padding: '0 10px 0 0' }}>
+														{sugg.source
+															&& sugg.source._recent_search
+															&& <CustomSvg
+																iconId={`${sugg.id}-icon`}
```

Comment:
@anik-ghosh-au7 There is no id property in the suggestion object. How did you test it?